DUI3-DX/Connectors/ArcGIS/Speckle.Connectors.ArcGIS3/Operations/Receive/HostObjectBuilder.cs

```diff
@@ -125,63 +231,75 @@ CancellationToken cancellationToken
 
     int allCount = objectsToConvert.Count;
     int count = 0;
-    Dictionary<string, (string path, Geometry converted, string? parentId)> convertedGeometries = new();
+    Dictionary<string, (string path, Geometry converted, string? parentId, Base baseObj)> convertedGeometries = new();
```

Comment:
what is the key string here and the value is too much. As far as understand `ConvertNonNativeGeometries` return 4 different value as tuple. We should start to get rid of this tuples.